src/coreclr/src/vm/processdiagnosticsprotocolhelper.cpp

```diff
@@ -108,12 +115,68 @@ bool ProcessInfoPayload::Flatten(BYTE * &lpBuffer, uint16_t &cbSize)
     if (fSuccess)
         fSuccess &= TryWriteString(lpBuffer, cbSize, Arch);
 
+    // Array<LPCWSTR>
```

Comment:
is this meant to be here?

Reply:
Yes, I put the type of the data I'm blitting above the code that blits it.